google/cloud/opentelemetry/quickstart/README.md

```diff
@@ -0,0 +1,227 @@
+
+### Testing
+
+The client library is only tested against the latest version of
+`opentelemetry-cpp`. That is v1.9.0, circa 2023-05.
+
+## Using with Bazel
+
+> :warning: If you are using Windows or macOS there are additional instructions
+> at the end of this document.
+
+1. Install Bazel using [the instructions][bazel-install] from the `bazel.build`
+   website.
+
+1. Compile this example using Bazel:
+
+   ```bash
+   cd $HOME/google-cloud-cpp/google/cloud/opentelemetry/quickstart
+   bazel build ...
+   ```
+
+   Note that Bazel automatically downloads and compiles all dependencies of the
+   project. As it is often the case with C++ libraries, compiling these
+   dependencies may take several minutes.
+
+1. Run the example, changing the placeholders to appropriate values:
+
+   ```bash
+   bazel run :quickstart -- [BUCKET_NAME] [PROJECT_ID]
+   ```
+
+### Details
+
+Note the following two feature flags explicitly set in the `.bazelrc`. Together,
+these flags enable OpenTelemetry tracing instrumentation in `google-cloud-cpp`.
+
+```py
+# Required for OpenTelemetry + Abseil compatibility
+build --@io_opentelemetry_cpp//api:with_abseil
+
+# Enables tracing instrumentation in google-cloud-cpp
+build --@google_cloud_cpp//:enable-experimental-opentelemetry
+```
+
+Also note that we explicitly load OpenTelemetry's dependencies in the
+`WORKSPACE.bazel`.
+
+## Using with CMake
+
+> :warning: If you are using Windows or macOS there are additional instructions
+> at the end of this document.
+
+### Package Managers
+
+`vcpkg` is the only package manager where `opentelemetry-cpp` is known to work
+alongside `abseil-cpp`.
+
+<!-- TODO(#11426): We need to add an opentelemetry feature to the vcpkg port
+after the next release. -->
+
+1. Install the dependencies with your favorite tools. As an example, if you use
+   [vcpkg](https://github.com/Microsoft/vcpkg.git):
+
+   ```bash
+   cd $HOME/vcpkg
+   ./vcpkg install google-cloud-cpp[core,trace,storage,experimental-opentelemetry]
+   ```
+
+   Note that, as it is often the case with C++ libraries, compiling these
+   dependencies may take several minutes.
+
+1. Configure CMake, if necessary, configure the directory where you installed
+   the dependencies:
+
+   ```bash
+   cd $HOME/google-cloud-cpp/google/cloud/opentelemetry/quickstart
+   cmake -H. -B.build -DCMAKE_TOOLCHAIN_FILE=$HOME/vcpkg/scripts/buildsystems/vcpkg.cmake
+   cmake --build .build
+   ```
+
+1. Run the example, changing the placeholders to appropriate values:
+
+   ```bash
+   .build/quickstart [BUCKET_NAME] [PROJECT_ID]
+   ```
+
+### Building from Source
+
+#### Building `google-cloud-cpp`
+
+The [packaging guide] contains instructions for how to build from source.
+
+The Google Cloud exporters are built as part of the feature,
+`experimental-opentelemetry`. This quickstart also uses the GCS client, from the
+`storage` feature.
+
+To enable these features, we build `google-cloud-cpp` with the following compile
```

Comment:
s/compile definition/cmake configuration option/

or maybe "add this to your CMake configuration command".

Reply:
Done.